src/codebook/nav/init.js

```diff
@@ -32,15 +33,6 @@ export function init(codebook) {
     t.wrap.classed('hidden', !t.active);
   });
 
```

Comment:
I think we need to keep the section below so that the boolean `controls` from  from available tabs is applied, and the controls are hidden in views where they don't do anything (settings and files).

Reply:
Just saw test notes in #240, which seem right to me. Maybe I was just missing part of the new logic to make this work without the deleted section?

Reply:
Seems ok. Approving the PR.